contracts/EasyAuction.sol

```diff
@@ -179,7 +180,7 @@ contract EasyAuction is Ownable {
             ),
             minimumBiddingAmountPerOrder,
             0,
-            bytes32(0),
+            IterableOrderedOrderSet.QUEUE_START,
```

Comment:
This is neat, but a little bit less gas efficient in most cases, where the interimOrder is not needed.